seatunnel-apis/seatunnel-api-flink/src/main/java/org/apache/seatunnel/flink/batch/FlinkBatchExecution.java

```diff
@@ -67,6 +67,7 @@ public void start(List<FlinkBatchSource> sources, List<FlinkBatchTransform> tran
             }
             input = transform.processBatch(flinkEnvironment, dataSet);
             registerResultTable(transform, input);
+            transform.registerFunction(flinkEnvironment);
```

Comment:
@simon824 What is the problem that this line of code is added to solve? If it's a bug, it's better to submit a separate PR to fix it, don't you think?

Reply:
The purpose of this pr is to fix the problem that udf is not registered, and this line is register flink batch udf, IMO we don't need create a new pr.